flambe/experiment/utils.py

```diff
@@ -145,6 +145,29 @@ def traverse(nested: Mapping[str, Any], path: Optional[List[str]] = None) -> Ite
             yield path, key, value
 
 
+def traverse_all(obj: Any) -> Iterable[Any]:
+    """Iterate over all nested mappings and sequences
+
+    Parameters
+    ----------
+    obj: Any
+
+    Returns
+    -------
+    Iterable[Any]
+        Iterable of child values to obj
+
+    """
+    if isinstance(obj, Mapping):
```

Comment:
I noticed that sets are not instances of Mapping or Sequence but they are instances of Iterable